esutil/bulk_indexer_internal_test.go

```diff
@@ -629,6 +630,111 @@ func TestBulkIndexer(t *testing.T) {
 			})
 		}
 	})
+
+	t.Run("MetaHeader presence in Request header", func(t *testing.T) {
+		type args struct {
+			disableMetaHeader bool
+			header            http.Header
+		}
+		tests := []struct {
+			name string
+			args args
+			want string
+		}{
+			{
+				name: "Meta header is present in header",
+				args: args{
+					disableMetaHeader: false,
+				},
+				want: `.*,h=bp`,
```

Comment:
Just verifying that there won't be a space between `.*` and `,`? I don't know how Go's newline folding works by default :)

Reply:
There shouldn't be, but I'll modify the test to exclude this possibility!